src/main.cpp

```diff
@@ -1,151 +1,119 @@
-        //     cancel_repeating_timer(&send_frame_rt);
-        //     // Arm motor
-        //     arm_motor();
-        //     // Re-enable timer
-        //     dma_alarm_rt_state = alarm_pool_add_repeating_timer_us(pico_alarm_pool, DMA_ALARM_PERIOD, repeating_send_dshot_frame, NULL, &send_frame_rt);
-        //     Serial.print("Re-enabled repeating DMA alarm: ");
-        //     Serial.println(dma_alarm_rt_state);
-
-        //     return;
-        // }
-
-        // b - beep
-        if (incomingByte == 98)
-        {
-            shoot::throttle_code = 1;
-            shoot::telemetry = 1;
-        }
+    printf("Setting up load cell\n");
+    hx711_t hx;
+    hx711_init(&hx, CLKPIN, DATPIN, pio0, &hx711_noblock_program,
+               &hx711_noblock_program_init);
+    hx711_power_up(&hx, hx711_gain_128);
+    hx711_wait_settle(hx711_rate_10);
 
-        // s - spin
-        // if (incomingByte == 115)
-        // {
-        //     // Disable timer
-        //     cancel_repeating_timer(&send_frame_rt);
-        //     // Ramp up to speed ARM_THROTTLE + 100
-        //     ramp_motor();
-        //     // Re-enable timer
-        //     dma_alarm_rt_state = alarm_pool_add_repeating_timer_us(pico_alarm_pool, DMA_ALARM_PERIOD, repeating_send_dshot_frame, NULL, &send_frame_rt);
-        //     Serial.print("Re-enabled repeating DMA alarm: ");
-        //     Serial.println(dma_alarm_rt_state);
-        // }
-
-        // r - rise
-        if (incomingByte == 114)
-        {
-            if (shoot::throttle_code >= ZERO_THROTTLE and shoot::throttle_code <= MAX_THROTTLE)
-            {
-                // Check for max throttle
-                if (shoot::throttle_code == MAX_THROTTLE)
-                {
-                    Serial.println("Max Throttle reached");
-                }
-                else
-                {
-                    shoot::throttle_code += 1;
-                    Serial.print("Throttle: ");
-                    Serial.println(shoot::throttle_code);
-                }
-            }
-            else
-            {
-                Serial.println("Motor is not in throttle mode");
-            }
-        }
+    printf("Setting up dshot\n");
+    tts::pwm_setup();
+    tts::dma_setup();
+    shoot::rt_setup();
 
-        // f - fall
-        if (incomingByte == 102)
-        {
-            if (shoot::throttle_code <= MAX_THROTTLE && shoot::throttle_code >= ZERO_THROTTLE)
-            {
-                if (shoot::throttle_code == ZERO_THROTTLE)
-                {
-                    Serial.println("Throttle is zero");
-                }
-                else
-                {
-                    shoot::throttle_code -= 1;
-                    Serial.print("Throttle: ");
-                    Serial.println(shoot::throttle_code);
-                }
-            }
-            else
-            {
-                Serial.println("Motor is not in throttle mode");
-            }
-        }
+    sleep_ms(1500);
```

Comment:
I put `sleep` before the print statements so that there is enough time for the serial to initialise

Reply:
Got it. I will move the sleep right after the led flashes.